src/network/networkd-network.c

```diff
@@ -929,6 +929,62 @@ int config_parse_ipv6_privacy_extensions(
         return 0;
 }
 
+static const char* const ipv6_accept_ra_table[_IPV6_ACCEPT_RA_MAX] = {
+        [IPV6_ACCEPT_RA_NO] = "no",
+        [IPV6_ACCEPT_RA_YES] = "yes",
+        [IPV6_ACCEPT_RA_KERNEL] = "kernel",
+};
+
+DEFINE_STRING_TABLE_LOOKUP(ipv6_accept_ra, IPv6AcceptRA);
```

Comment:
Please use `DEFINE_STRING_TABLE_LOOKUP_WITH_BOOLEAN()` macro. Then, `config_parse_ipv6_accept_ra()` can be defined through `DEFINE_CONFIG_PARSE_ENUM()` macro.